deploy/elasticsearch/templates/clusterdefinition.yaml

```diff
@@ -0,0 +1,223 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ClusterDefinition
+metadata:
+  name: elasticsearch
+  labels:
+    {{- include "elasticsearch.labels" . | nindent 4 }}
+spec:
+  type: elasticsearch
+  connectionCredential:
+    username: root
+    password: "$(RANDOM_PASSWD)"
+    endpoint: "https://$(SVC_FQDN):$(SVC_PORT_http)"
+    host: "$(SVC_FQDN)"
+    port: "$(SVC_PORT_http)"
```

Comment:
port with the name `http` is not found in the `componentDefs.service.ports`, should it be `tcp-rest-api`?

Reply:
please check if the `connectionCredential`  is correctly rendered.